CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ configure_package_config_file(cmake/ArborXConfig.cmake.in
 )
 write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/ArborXConfigVersion.cmake
   VERSION ${ARBORX_VERSION_STRING}
-  COMPATIBILITY AnyNewerVersion
+  COMPATIBILITY SameMajorVersion
```

Comment:
Please confirm that we are not adding a version range argument

Reply:
> Please confirm that we are not adding a version range argument

What would a range argument to `SameMajorVersion` mean?

Reply:
I think I misunderstood how this works.  The version range is at find_package() call site in user code.